lib/dor/models/describable.rb

```diff
@@ -241,8 +241,9 @@ def to_solr(solr_doc = {}, *args)
           solr_doc[solr_key].push *vals unless vals.nil? || vals.empty?
           # asterisk to avoid multi-dimensional array: push values, not the array
         end
-        solr_doc['sw_pub_date_sort_ssi' ] = mods.pub_date_sort  # e.g. '0800'
-        solr_doc['sw_pub_date_facet_ssi'] = mods.pub_date_facet # e.g. '9th century'
+        solr_doc['sw_pub_date_sort_ssi' ] = mods.pub_year_sort_str  # e.g. '0800'
+        solr_doc['sw_pub_date_sort_isi' ] = mods.pub_year_int  # e.g. '0800'
+        solr_doc['sw_pub_date_facet_ssi'] = mods.pub_year_display_str # e.g. '9th century'
```

Comment:
i would be open to renaming the solr field names to match the new stanford-mods method names, and updating references in argo.  opinions on that?